src/components/EditorWrapper.vue

```diff
@@ -318,6 +322,10 @@ export default {
 				},
 			}
 		},
+		saveStatusClass() {
+			if (this.syncError && this.lastSavedString !== '') return 'save-error'
+			return this.dirtyStateIndicator ? 'saving-status' : 'saved-status'
+		},
```

Comment:
We can turn it more readable.
```suggestion
		saveStatusClass() {
			if (this.syncError && this.lastSavedString !== '') {
			  return 'save-error'
			 }
			return this.dirtyStateIndicator ? 'saving-status' : 'saved-status'
		},
```